aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -247,18 +238,12 @@ pub fn validate_history(
         if history.min_overflow_positive_delta.is_some()
             && base_value <= max_value - history.min_overflow_positive_delta.unwrap()
         {
-            return Err(abort_error(
-                format!("Overflow was expected when setting the aggreagator start value to {}. Previous speculative start value = {:?}, Min overflow delta = {}, Max value = {}", base_value, speculative_start_value, history.min_overflow_positive_delta.unwrap(), max_value),
-                EEXPECTED_OVERFLOW,
-            ));
+            return Err(BoundedMathError::Overflow);
```

Comment:
When validating history, this is a scenario where the original execution went well, but an overflow is supposed to occur. I distinguish these EXECPECTED_OVERFLOW, EXPECTED_UNDERFLOW errors from the original ADD_OVERFLOW, SUB_UNDERFLOW errors. Distinguishing the error codes might help us when debugging. Do we wanna merge those error codes?

Reply:
yeah, I don't know what's better. 

but I can revert change to validate_history, as that doesn't affect anything else anyways

Reply:
updated